packages/now-node/src/index.ts

```diff
@@ -470,31 +471,47 @@ export async function startDevServer(
 async function doTypeCheck({
   entrypoint,
   workPath,
+  meta = {},
 }: StartDevServerOptions): Promise<void> {
-  // In order to type-check a single file, a temporary tsconfig
+  const { devCacheDir = join(workPath, '.vercel', 'cache') } = meta;
```

Comment:
```suggestion
  const { devCacheDir = join(workPath, '.now', 'cache') } = meta;
```

Actually lets use  `.now` here since the only time `devCacheDir` will be undefined is for a old CLI. This will match now-node and now-python fallback behavior.

New versions of the CLI will use `.vercel` already.

Reply:
Sure, makes sense. However, the `startDevServer()` function will never be executed using one of those old CLI's. This fallback is mostly to appease TypeScript.